packages/eslint-plugin/tests/rules/strict-boolean-expressions.test.ts

```diff
@@ -925,6 +954,24 @@ ruleTester.run('strict-boolean-expressions', rule, {
         },
       ],
     },
+    {
+      errors: [
+        {
+          messageId: 'strictBooleanExpression',
+          line: 2,
+          column: 55,
+        },
+        {
+          messageId: 'strictBooleanExpression',
+          line: 3,
+          column: 37,
+        },
+      ],
+      code: `
+        const f1 = (x: boolean | null | undefined) => x ? 1 : 0;
+        const f2 = (x?: boolean) => x ? 1 : 0;
```

Comment:
can you add test case without any option set?
```
const f1 = (x: boolean | {}) => x ? 1 : 0;
```

Reply:
https://github.com/typescript-eslint/typescript-eslint/pull/1385/commits/cb3e8b575a179d06fa6129106201df3bdff4c741
thanks! (didn't know what yellow means in the codecov page)

Reply:
it means that you missed branch / else case